docs/documentation/securing_apps/topics/saml/java/java-adapters.adoc

```diff
@@ -2,3 +2,47 @@
 === {project_name} Java adapters
 
 {project_name} comes with a range of different adapters for Java application. Selecting the correct adapter depends on the target platform.
+
+include::general-config.adoc[]
+include::general-config/sp_element.adoc[]
+include::general-config/sp-keys.adoc[]
+include::general-config/sp-keys/keystore_element.adoc[]
+include::general-config/sp-keys/key_pems.adoc[]
+include::general-config/sp_principalname_mapping_element.adoc[]
+include::general-config/roleidentifiers_element.adoc[]
+include::general-config/sp_role_mappings_provider_element.adoc[]
+include::general-config/idp_element.adoc[]
+include::general-config/idp_allowedclockskew_subelement.adoc[]
+include::general-config/idp_singlesignonservice_subelement.adoc[]
+include::general-config/idp_singlelogoutservice_subelement.adoc[]
+include::general-config/idp_keys_subelement.adoc[]
+include::general-config/idp_httpclient_subelement.adoc[]
+include::saml-jboss-adapter.adoc[]
+include::jboss-adapter/jboss_adapter_installation.adoc[]
+include::jboss-adapter/jboss-adapter-samesite-setting.adoc[]
+
+ifeval::[{project_product}==true]
+include::jboss-adapter/jboss-adapter-rpms.adoc[]
```

Comment:
I guess this can be removed too, if we're referencing documentation from RH-SSO.

Reply:
Left the file there even though it's not actually included in the product builds. Reasoning for leaving it was to have a small change in the PR, but also it can maybe be useful when we add docs for EAP 8 later as that will be distributed as RPMs.